about.html

```diff
@@ -0,0 +1,150 @@
+              <li><a href="#" class="sub_nav_link">Program</a></li>
+              <li><a href="#third_section" class="sub_nav_link">Speakers</a></li>
+              <li><a href="#" class="sub_nav_link">Join</a></li>
+            </ul>
+        </div>
+
+        <div class="sub_nav_container">
+          <a href="#" class="startup_logo"><img class="img_logo" src="capstone-image/new logo.png" alt="startup"></a>
+
+          <div class="mobile_menu_container">
+            <ul class="mobile_nav_link">
+              <li><a href="index.html" class="mobile_nav_link">Home</a></li>
+              <li><a href="#" class="mobile_nav_link">Program</a></li>
+              <li><a href="#" class="mobile_nav_link">Join</a></li>
+              <li><a href="#third_section" class="mobile_nav_link">Speakers</a></li>
+              <li><a href="#" class="mobile_nav_link">News</a></li>
+            </ul>
+
+            <button class="nav_button" type="button">ST Campaign</button>
+          </div>
+        </div>
+
+        <div class="hamburger_menu">
+          <span class="ham_bar"></span>
+          <span class="ham_bar"></span>
+          <span class="ham_bar"></span>
+        </div>
+
+      </nav>
+    </header>
+     <div class="main_section_container">
+            <div class="top_section_container">
+            <p class="main_section_title">"Join Our Innovation!!"</p>
+            <h1 class="theme">MEET-UP TO START UP, 2023 TECH CONFERENCE!</h1>
+            </div>
+
+           <div class="main_section_content">
+             <p class="main_section_text">
+              Regie is designed to produce sales and marketing copy for brands to employ more quickly and yet with more consistency and personalization than doing so manually. The startup adapts the GPT-3 model specifically for businesses to use in email campaigns, online ads, text messages, social media, and even podcast ad copy. Keeping up with the demand for all of that content across so many media becomes a lot easier when AI can take a phrase or two as suggestions and rapidly compose multiple messages for each outlet and style of writing after the startup trained its GPT-3 engine on 20,000 sales sequences. Clients can integrate Regie into their own content management systems or use it a browser extension, and the company claims to track thousands of sales sequences and more than 35 million emails. It’s already used by more than 75 brands, including Crunchbase, Hubspot, and AT&T, though the content is checked by a human curator to avoid any issues of unsupervised AI content generation.</p>
+
+              <p class="contact_us">
+                Please contact us now per Email to get started if you are ready join hundreds of entreprenuers for this edition of "MEET-UP TO START UP"
+              </p>
+              <a href="mailto:info@decagoninstitute.com" class="email_add">info@decagoninstitute.com</a>
+
+           </div>
+     </div>
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/3nityGitHub/CAPSTONE_PROJECT_1/pull/1#discussion_r1179531812), please make sure to align the text to the center and use the same background color for the heading ⚒ The  paragraph or `p` tag is not aligned to the center as given in the design guidelines. See the images below for more clarity 👇
- [ ] Also, please fix the heading. I understand that you are trying to use `background-clip` property, but it is not properly implemented. Please fix this 📌You can refer to the link mention in the above comment for it 👆

Your design previously | Your design now 
---| ---|
![image](https://user-images.githubusercontent.com/106119105/235029648-b5576565-02e9-458f-b89c-6b67423fc392.png) | ![image](https://user-images.githubusercontent.com/106119105/235029617-10144ed0-017a-4d36-a61c-20f3a28c43ce.png)